js/private/js_run_devserver.mjs

```diff
@@ -284,30 +423,32 @@ async function main(args, sandbox) {
     })
 }
 
-;(async () => {
-    let sandbox
-    try {
-        sandbox = path.join(
-            await fs.promises.mkdtemp(
-                path.join(os.tmpdir(), 'js_run_devserver-')
-            ),
-            process.env.JS_BINARY__WORKSPACE
-        )
-        // Intentionally synchronous; see comment on mkdirpSync
-        mkdirpSync(path.join(sandbox, process.env.JS_BINARY__CHDIR || ''))
-        await main(process.argv.slice(2), sandbox)
-    } catch (e) {
-        console.error(e)
-        process.exit(1)
-    } finally {
+if (!process.env.__RULES_JS_UNIT_TEST__) {
```

Comment:
Can we do `if (process.env.__RULES_JS_UNIT_TEST__) return` or something like that to avoid indenting this entire thing?

Reply:
Can't think of how since this is evaluated during module loading

Reply:
Actually, there I can just short-circuit out of the async function 👍